Tweak.xm

```diff
@@ -263,13 +264,30 @@ static void activatePiP(YTPlayerPIPController *controller) {
         MLPIPController *pip = [controller valueForKey:@"_pipController"];
         if ([controller canEnablePictureInPicture])
             [pip activatePiPController];
-        else
-            [pip deactivatePiPController];
+        // else
+        //     [pip deactivatePiPController];
     }
 }
 
 %hook YTPlayerViewController
 
+- (id)initWithParentResponder:(id)arg1 overlayFactory:(id)arg2 {
+    %orig;
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.1 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
```

Comment:
Can you confirm this works 100% of the time you want to PiP any videos?

Reply:
> Can you confirm this works 100% of the time you want to PiP any videos?

On my device, it never failed once. Might be worth testing it on older devices to make sure.